### DISCUSSION TITLE
How to disable this tab without disabling plugin?

### DISCUSSION THREAD

**Discussion by dalandanelie** [likehopper/redmine_sla#19] [Q&A]
hi! I just wanted to ask how to remove the SLA tab on the project menu when plugin is enabled? doesn't do much for me with my current setup. Thankyou.
![image](https://github.com/user-attachments/assets/2698b6e1-5a2f-49f1-bd4d-93fd2043cd55)

**Comment by likehopper** [comment]
The SLA tab is displayed :
- if the module is enabled on the project
[ and ]
- if the user has an 'SLA manage' role on the project (or if they are an administrator)."

It's okay for you ?

**Reply by dalandanelie** [reply]
Its fine, but when i click the SLA tab it only shows 404. It kind of bugs me that it does not point to anything when clicked, any suggestions on those?

**Reply by likehopper** [reply]
Sorry, I have not been able to reproduce this issue. 
Could you increase log level to send me your debug log file with this 404 ? 
Is it possible for you to do a screenshot of admin/info ? 
And I'm interested in any clue that will help me to solve your problem.

**Reply by dalandanelie** [reply]
I do not know where the logs are but here is the error.
![image](https://github.com/user-attachments/assets/7675d2aa-72e1-4b6e-8e90-fe14a36fac92)
When i try to click the SLA tab from the project dahsboard it returns a 404.

**Reply by likehopper** [reply]
Hi, 

Is it possible for you to do a screenshot of admin/info ? And also the plugin's configuration in the project?

For the logs, go to the folder where the project is installed. In the "config/application.rb" file, you have to look for the "config.log_level" directive and force it with the value ":debug" like this: `config.log_level = :debug`

Once Redmine is restarted, you should have a log file according to your environment in the "log/" folder.

Thanks
